geode-core/src/main/java/org/apache/geode/internal/cache/PRHARedundancyProvider.java

```diff
@@ -1712,8 +1712,7 @@ private PartitionedRegion findPersistentRegionRecursively(PartitionedRegion part
   }
 
   void scheduleCreateMissingBuckets() {
-    if (partitionedRegion.getColocatedWith() != null
-        && ColocationHelper.isColocationComplete(partitionedRegion)) {
+    if (partitionedRegion.getColocatedWith() != null) {
```

Comment:
PRHARedundancyProviderTest should be updated with a test that fails without this change.